web/ci.sh

```diff
@@ -195,28 +190,33 @@ if builder_start_action prepare:downloads.keyman.com; then
 
   pushd build/publish
   # Zip both the 'debug' and 'release' configurations together.
-  "${COMPRESS_CMD}" $COMPRESS_ADD $ZIP *
+  "${COMPRESS_CMD}" ${COMPRESS_ADD} ${ZIP }*
```

Comment:
What effect does the extra space here (for `${ZIP }`) have?

Reply:
Oops, the closing brace landed in the wrong spot. Fixed.